README.md

```diff
@@ -158,4 +164,3 @@ end
 Then when we call `bub.birthday!`, Lil Bub's age becomes 3. Happy birthday, Bub!
 
 ![bub birthday](http://ihavecat.com/wp-content/uploads/2014/06/480ae292-8e53-41ff-9b5a-868ef5f75d42.jpg)
```

Comment:
we don't want to hotlink to another persons images. What if they change their links or their website goes down? Use [this guide](https://github.com/learn-co-curriculum/learn-readme-style-guide#amazon-s3-1) on how to upload images to s3